### DISCUSSION TITLE
curl http://192.168.136.77:64367/receive/mnj4 download failed?

### DISCUSSION THREAD

**Discussion by YANGJINJUE** [claudiodangelis/qr-filetransfer#324] [Q&A]
curl http://192.168.136.77:64367/receive/mnj4 download failed? why? how to solve the problem?
chrome download success。

**Comment by YDX-2147483647** [comment]
Maybe `qrcp send …` will work? Your URL is `/receive/`, but `/send/` is desired.

**Comment by claudiodangelis** [comment]
Hello everyone, sorry for the huge delay.
@YDX-2147483647 is correct, that URL is generated when you want to send a file from the mobile to the laptop, so it's supposed to be open with a web browser on the mobile.

If you want to download a file using `curl`, you need to "send" the file, for example: `qrcp your-file.txt`.

Please let me know if this helps.
Thanks!
C
